.github/workflows/kind-e2e.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         with:
           version: 'v1.19.0'
       - name: Setup kind
-        uses: engineerd/setup-kind@v0.4.0
+        uses: engineered/setup-kind@v0.4.0
```

Comment:
/hold

This is a bad rename, we should add this repo to the prettier-exclude.yaml in knative-sandbox/knobots

Reply:
oh, damn